### PR TITLE
Fixed miner test values, gave constants for less error-prone values.

### PR DESCRIPTION
**Pull request by instagibbs**
Based on discussion in #7784 I decided to do a bit more cleanup.

### CODE REVIEW

**Comment by instagibbs** [comment]
Doing this I noticed yet another mis-typed value on https://github.com/bitcoin/bitcoin/pull/7807/files#diff-5c64500485fda76388a86c95c0059585L173 , so it seemed worthwhile.

**Comment by MarcoFalke** [comment]
That's clearer. Thanks. utACK 11679ca

**Comment by laanwj** [comment]
utACK

**Comment by btcdrak** [comment]
utACK

File: src/test/miner_tests.cpp [hunk 1]
```diff
@@ -167,13 +172,13 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
         tx.vin[0].scriptSig << vchData << OP_DROP;
     tx.vin[0].scriptSig << OP_1;
     tx.vin[0].prevout.hash = txFirst[0]->GetHash();
-    tx.vout[0].nValue = 5000000000LL;
+    tx.vout[0].nValue = BLOCKSUBSIDY;
     for (unsigned int i = 0; i < 128; ++i)
     {
-        tx.vout[0].nValue -= 10000000;
```
Comment (paveljanik) [inline src/test/miner_tests.cpp]: 10*LOWFEE?

File: src/test/miner_tests.cpp [hunk 2]
```diff
@@ -211,7 +216,7 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
     tx.vout[0].nValue = 0;
     hash = tx.GetHash();
     // give it a fee so it'll get mined
-    mempool.addUnchecked(hash, entry.Fee(100000).Time(GetTime()).SpendsCoinbase(false).FromTx(tx));
```
Comment (paveljanik) [inline src/test/miner_tests.cpp]: LOWFEE/10?

**Comment by paveljanik** [comment]
As you are not replacing constants only, can you please change it so that you first replace constants unchanged and then do the changes, ie two commits? It will make review much easier...

**Comment by instagibbs** [comment]
@paveljanik done

**Comment by paveljanik** [comment]
@instagibbs Reading this thoroughly, I think we could also define constants in the term of COIN and CENT, it could be even more descriptive.

**Pull request merged**